cmd/internal/gengo/go.go

```diff
@@ -73,10 +73,11 @@ var Cmd = &base.Command{
 }
 
 var (
-	flag      = &Cmd.Flag
-	flagDebug = flag.Bool("debug", false, "set log level to debug")
-	flagTest  = flag.Bool("test", false, "test Go+ package")
-	flagSlow  = flag.Bool("slow", false, "don't cache imported packages")
+	flag        = &Cmd.Flag
+	flagDebug   = flag.Bool("debug", false, "set log level to debug")
+	flagTest    = flag.Bool("test", false, "test Go+ package")
+	flagSlow    = flag.Bool("slow", false, "don't cache imported packages")
+	flagRebuild = flag.Bool("a", false, "force rebuilding of packages that are already up-to-date")
```

Comment:
Why is this flag 'a', other commands are 'rebuild'?

Reply:
fixed